specifyweb/frontend/js_src/lib/components/Formatters/Fields.tsx

```diff
@@ -54,7 +54,7 @@ export function Fields({
            *   table layout with list layout
            */
           className={`
-            grid-table min-w-[35rem] gap-x-4 gap-y-4
+            grid-table min-w-[35rem] gap-x-2 gap-y-2
```

Comment:
minor thing, but just a fyi that this is equivalent:

```suggestion
            grid-table min-w-[35rem] gap-2
```